.github/workflows/libclang-abi-tests.yml

```diff
@@ -51,9 +51,10 @@ jobs:
         id: vars
         run: |
           remote_repo='https://github.com/llvm/llvm-project'
-          if [ ${{ steps.version.outputs.LLVM_VERSION_MINOR }} -ne 0 ] || [ ${{ steps.version.outputs.LLVM_VERSION_PATCH }} -eq 0 ]; then
+          echo "BASELINE_VERSION_MINOR=1" >> "$GITHUB_OUTPUT"
```

Comment:
It doesn't seem like this variable ends up getting used anywhere? It does in `llvm-tests`, but down below only `BASELINE_REF` seems to be used?

Seems like `BASELINE_REF` isn't used in `llvm-tests.yml` and can probably be removed...